dot/core/digest.go

```diff
@@ -180,14 +181,22 @@ func (h *DigestHandler) handleBABEChangesOnImport(header *types.Header) {
 	num := header.Number
 	resume := h.babeResume
 	if resume != nil && num.Cmp(resume.atBlock) == 0 {
-		h.babe.SetAuthorities(h.babeAuths)
+		err := h.babe.SetAuthorities(h.babeAuths)
+		if err != nil {
+			log.Error("error setting authorities", "error", err)
+			return
```

Comment:
I think you can change this to a `warn` and not return, since it's ok to update the authorities to a set that doesn't include our key